.readme-partials.yaml

```diff
@@ -1,51 +1,42 @@
+  ### Creating a new subscription or using an existing subscription
 
-                  Follow [these instructions](https://cloud.google.com/pubsub/lite/docs/quickstart#before-you-begin).
+   Follow [the instruction](https://cloud.google.com/pubsub/lite/docs/quickstart#create_a_lite_subscription) to create a new subscription or use an existing subscription. If using an existing subscription, the connector will read from the oldest unacknowledged message in the subscription.
 
-  ### Create a new subscription or use existing subscription
+  ### Creating a Google Cloud Dataproc cluster (Optional)
 
-                  Follow [the instruction](https://cloud.google.com/pubsub/lite/docs/quickstart#create_a_lite_subscription) to create a new
-                  subscription or use existing subscription. If using existing subscription, the connector will read message from the
-                  oldest unacknowledged.
+   If you do not have an Apache Spark environment, you can create a [Cloud Dataproc](https://cloud.google.com/dataproc/docs) cluster with pre-configured auth. The following examples assume you are using Cloud Dataproc, but you can use `spark-submit` on any cluster.
 
-  ### Create a Google Cloud Dataproc cluster (Optional)
-
-                  If you do not have an Apache Spark environment you can create a Cloud Dataproc cluster with pre-configured auth. The following examples assume you are using Cloud Dataproc, but you can use `spark-submit` on any cluster.
-
-                  ```
-                  MY_CLUSTER=...
-                  gcloud dataproc clusters create "$MY_CLUSTER"
-                  ```
+    ```
+    MY_CLUSTER=...
+    gcloud dataproc clusters create "$MY_CLUSTER"
+    ```
 
   ## Downloading and Using the Connector
 
   <!--- TODO(jiangmichael): Add jar link for spark-pubsublite-latest.jar -->
-    The latest version connector of the connector (Scala 2.11) is publicly available in
-    gs://spark-lib/pubsublite/spark-pubsublite-latest.jar.
+    The latest version connector of the connector (Scala 2.11) will be publicly available in `gs://spark-lib/pubsublite/spark-pubsublite-latest.jar`.
 
   <!--- TODO(jiangmichael): Release on Maven Central and add Maven Central link -->
-                              The connector is also available from the Maven Central
-                              repository. It can be used using the `--packages` option or the
-                              `spark.jars.packages` configuration property. Use the following value
+    The connector will also be available from the Maven Central repository. It can be used using the `--packages` option or the `spark.jars.packages` configuration property.
 
-                              | Scala version | Connector Artifact |
-                              | --- | --- |
-                              | Scala 2.11 | `com.google.cloud.pubsublite.spark:pubsublite-spark-sql-streaming-with-dependencies_2.11:0.1.0` |
+    | Scala version | Connector Artifact |
```

Comment:
maybe comment the table out and add back once we actually uploaded to maven?

Reply:
done.